core/iwasm/common/wasm_runtime_common.h

```diff
@@ -80,6 +85,7 @@ STORE_U8(void *addr, uint8_t value)
 #define LOAD_U32(addr) (*(uint32 *)(addr))
 #define LOAD_I16(addr) (*(int16 *)(addr))
 #define LOAD_U16(addr) (*(uint16 *)(addr))
+#define LOAD_V128(addr) (*(V128 *)(addr))
```

Comment:
Had better also define STORE_V128 and LOAD_V128 when WASM_CPU_SUPPORTS_UNALIGNED_ADDR_ACCESS isn't 1.
see L95 to L386.